commands/enrich/enrich.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
 	"github.com/jfrog/jfrog-client-go/utils/log"
 	"github.com/jfrog/jfrog-client-go/xray/services"
+	orderedJson "gitlab.com/c0b/go-ordered-json"
```

Comment:
can you use github.com/virtuald/go-ordered-json instead?

also - please keep the breaklines in the GO - so all of the jfrog ones will be together, all the github ones if possible (I see "github.com/jfrog/jfrog-cli-security/utils/results/output" was moved unnecessarily

Reply:
changed package but order is predetermined